server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
FWIW, I modified the strings in question here so the substrings are `expires in` `x days` rather than 'expires``in x days`  since `expires in` is a single part of speech (compound verb). Seems like this should make things easier to localize.